OrbitGl/TimeGraph.cpp

```diff
@@ -583,7 +589,12 @@ void TimeGraph::UpdatePrimitives(bool a_Picking) {
 
           col = isSelected
```

Comment:
this is a lot for :? maybe replace with if/else if else?

Reply:
Done.